osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -370,22 +406,48 @@ private void beatmapsChanged(IRealmCollection<BeatmapInfo> sender, ChangeSet? ch
             if (selectedBeatmapSet?.BeatmapSet.ID == beatmapSet.ID)
                 previouslySelectedID = selectedBeatmap?.BeatmapInfo.ID;
 
-            var newSet = createCarouselSet(beatmapSet);
-            var removedSet = root.RemoveChild(beatmapSet.ID);
+            var removedSets = root.RemoveChild(beatmapSet.ID);
```

Comment:
The fact that `RemoveChild()` returns multiple items is a bit weird now... That said not sure what would be a better name, either. `RemoveByID()`?

Reply:
Renamed to `RemoveItemsByID`.